_data/glossary/en/sighash-single.yaml

```diff
@@ -7,9 +7,10 @@ required:
     title_max_40_characters_no_formatting: SIGHASH_SINGLE
 
     summary_max_255_characters_no_formatting: >
-        Signature hash type which only signs its input and the output
-        with the same index value, allowing modification of other inputs
-        and outputs.
+        Signature hash type which signs the output corresponding to this input
+        (the one with the same index value), this input, and any other inputs
+        partially. Allows modification of other outputs. The partial signing of
+        other inputs allows modification of their sequence numbers.
```

Comment:
@mruddy this text is good, but is there any chance you can edit it down to 255 characters or less?  I think dropping the final sentence would be fine since it links to the full description.